package.json

```diff
@@ -34,7 +34,7 @@
     "@types/body-parser": "1.19.0",
     "@types/classnames": "2.2.10",
     "@types/cors": "2.8.6",
-    "@types/express": "4.17.6",
+    "@types/express": "^4.17.1",
```

Comment:
`@types/express` issue should be fixed in master.
Can you please try with a fresh copy of repo and if it works remove this change from PR?

Reply:
Will close this one and create new PR to get clear history